test/parsing-test.js

```diff
@@ -12,13 +12,20 @@ let helper = new Helper('../scripts/main.coffee')
 
 describe('Messages parsing', function () {
     let msg
+    let room
+    let a
+    let b
```

Comment:
`a` and `b`?? not really  getting what are these - Gdocs and  Gist? can we have more reasonable names for variables?